UI/Pages/SearchResults.cshtml

```diff
@@ -7,131 +7,131 @@
-					<govuk-button type="submit" data-testid="call-to-action">Search</govuk-button>
-				</div>
-			</div>
-
-			<div show-if="Model.Data.Results != null" class="govuk-body" data-testid="results-summary">
-				<hr class="govuk-section-break govuk-section-break--m govuk-section-break--visible">
-				@{
-					var count = Model.Data.Results!.Count == 0 ? "No" : Model.Data.Results.Count.ToString();
-					var resultPlural = Model.Data.Results.Count != 1 ? "results" : "result";
-				}
-				<span class="govuk-!-font-size-24"><strong>@count</strong> @(resultPlural) <span show-if="@Model.Data.LocalAuthority != null">for <strong>@Model.Data.LocalAuthority</strong></span></span><br/>
-				Your results are in a random order.
-				<hr class="govuk-section-break govuk-section-break--m govuk-section-break--visible">
-			</div>
-
-
-			@if (Model.Data.Results != null)
-			{
-				foreach (var item in Model.Data.Results.Results)
-				{
-					<div data-testid="results-list-item">
-						<h2 class="govuk-heading-m"><a asp-page="TuitionPartner" asp-route-id="@item.SeoUrl" class="govuk-link">@item.Name</a></h2>
-						<govuk-summary-list>
-							<govuk-summary-list-row class="govuk-body-s">
-								<govuk-summary-list-row-key>
-									Subjects covered
-								</govuk-summary-list-row-key>
-								<govuk-summary-list-row-value>
-									<ul class="govuk-list govuk-body-s">
-									@foreach (var keyStageSubjects in item.Subjects.GroupBy(e => e.KeyStageId))
-									{
-										<li>@(((KeyStage)keyStageSubjects.Key).DisplayName()) - @keyStageSubjects.DisplayList()</li>
-									}
-									</ul>
-								</govuk-summary-list-row-value>
-							</govuk-summary-list-row>
-							<govuk-summary-list-row class="govuk-body-s">
-								<govuk-summary-list-row-key>
-									Type of tuition
-								</govuk-summary-list-row-key>
-								<govuk-summary-list-row-value>
-									@string.Join(", ", item.TuitionTypes.Select(e => e.Name))
-								</govuk-summary-list-row-value>
-							</govuk-summary-list-row>
-							<govuk-summary-list-row class="govuk-body-s">
-								<govuk-summary-list-row-key>
-									SEND support
-								</govuk-summary-list-row-key>
-								<govuk-summary-list-row-value>
-									@item.HasSenProvision.ToYesNoString()
-								</govuk-summary-list-row-value>
-							</govuk-summary-list-row>
-							<govuk-summary-list-row class="govuk-body-s">
-								<govuk-summary-list-row-key>
-									Tuition partner information
-								</govuk-summary-list-row-key>
-								<govuk-summary-list-row-value>
-									@item.Description
-								</govuk-summary-list-row-value>
-							</govuk-summary-list-row>
-						</govuk-summary-list>
-					</div>
-				}
-			}
-		</div>
-	
-	</form>
+						Model.Data.AllSubjects[item].Select(x => ($"{item}-{x.Name}".ToSeoUrl(), $"{item}-{x.Name}", x.Name, x.Selected)).OrderBy(x=> x.Name)
+						)) " />
+                }
+
+                <govuk-radios asp-for="Data.TuitionType" class="govuk-radios--small">
```

Comment:
Adding class `govuk-!-margin-bottom-3` will get this closer to the spacing of the design

Reply:
Added class